blockchain/blockdao.go

```diff
@@ -521,6 +523,11 @@ func (dao *blockDAO) putBlock(blk *Block) error {
 		batch.Put(blockNS, topHeightKey, height, "failed to put top height")
 	}
 
+	// only log this info if you enable explorer
```

Comment:
// only build Tsf/Vote/Execution index if enable explorer?
and move this comment to line 531 below?